CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 ### Performance enhancements
 
 ### Bug fixes
+* `skbio.stats.composition.ancom` is compatible with scipy v0.18 and numpy v1.12.0 ([#1472](https://github.com/biocore/scikit-bio/issues/1472)).
```

Comment:
All I did is replace the np.apply_along_axis with list comprehensions.  This was because np.apply_along_axis is having some weird type handling in the newest version of numpy.  What sort of statement do you think would be appropriate to add to the changelog?